package.json

```diff
@@ -80,6 +84,7 @@
     "project-update": "pip install --upgrade pip && npm update",
     "test": "npm run test:js && npm run test:python",
     "test:js": "node ./tests/functional/_intern.js",
+    "test:unit-js": "intern config=./webcompat/static/js/lib/wizard/tests/intern.json",
```

Comment:
We have another intern config file (for the functional tests) at `tests/functional/_intern.js` -- but this one is relative to the wizard lib sources. Is that a convention that folks follow, or should we consider moving those tests to the top level tests directory? I defer to you!

(to be clear, i'm not suggesting we try to combine these intern configs)

Reply:
yeah, I wasn't sure about this :) but find it more convenient to have unit tests where the code is and tend to see that in other projects as well

Reply:
Cool, let's keep it for now -- easy enough to move later on if we feel strongly about it. At some point we'll want to update the testing docs so people can know where it is, etc.